client/src/components/ratings-reviews/Index.jsx

```diff
@@ -1,14 +1,77 @@
 import React from 'react';
+// import reviews from '../../../../sampleData/reviews.js';
+import { useDispatch, useSelector } from 'react-redux';
+import { useEffect } from 'react';
+import axios from 'axios';
+import API_KEY from '../../config/config.js';
+import { Container, Row, Col } from 'react-bootstrap';
+
+
+
 
 ////////////////////////////////////////////////////////////////////////////////////////////////
 // Import modules to use in component creation, uncomment as neccessary
 ////////////////////////////////////////////////////////////////////////////////////////////////
 
-// import ReviewsList from './ReviewsList.jsx';
-// import IndividualReviewTile from './IndividualReviewTile.jsx';
+import ReviewsList from './ReviewsList.jsx';
+import IndividualReviewTile from './IndividualReviewTile.jsx';
 // import Sort from './Sort.jsx';
-// import RatingBreakdown from './RatingBreakdown.jsx';
+import RatingBreakdown from './RatingBreakdown.jsx';
 // import Recommendations from './Recommendations.jsx';
 // import ProductBreakdown from './ProductBreakdown.jsx';
 // import NewReview from './new-review/Index.jsx';
-// import KeywordSearch from './KeywordSearch.jsx';
\ No newline at end of file
+// import KeywordSearch from './KeywordSearch.jsx';
+
+// infinity stone product_id 16065
+
+
+
+const RatingsReviews = () => {
+  let dispatch = useDispatch();
+  const sortType = useSelector(state => state.reviewsSortType)
+  const productId = useSelector(state => state.productId)
+  console.log('sortType', sortType)
+
+  useEffect(() => {
```

Comment:
You may want to write a function that can handle both of these so that your code is a little more DRY. Maybe a function that sends a get request, and you can pass in the URL, params, and dispatch methods? I did something like this:

`  const fetchItem = (action, link) => {
    axios.get(link, {headers: {Authorization: API_KEY}})
    .then(response => {
      dispatch(action(response.data));
    })
    .catch(err => { console.log(err) })
  }`